src/main/java/org/tb/persistence/EmployeeDAO.java

```diff
@@ -223,4 +229,35 @@ public boolean deleteEmployeeById(long emId) {
         return false;
     }
 
+
```

Comment:
HttpServletRequest hat in einem DAO mMn nichts zu suchen. Ein DAO regelt den Zugriff auf den persistenten Speicher.